src/main/io/gps.c

```diff
@@ -38,6 +38,8 @@
 
 #include "sensors/sensors.h"
 
+#include "io/cms.h"
```

Comment:
display.h included few lines below has declaration of displayCmsInit(displayPort_t *pPort).